TM1py/Services/CellService.py

```diff
@@ -1062,16 +1134,18 @@ def execute_mdx_ui_array(
         :param elem_properties: List of properties to be queried from the elements. E.g. ['UniqueName','Attributes']
         :param member_properties: List of properties to be queried from the members. E.g. ['UniqueName','Attributes']
         :param value_precision: Integer (optional) specifying number of decimal places to return
+        :param sandbox_name: str
         :return: dict :{ titles: [], headers: [axis][], cells:{ Page0:{ Row0:{ [row values], Row1: [], ...}, ...}, ...}}
         """
-        cellset_id = self.create_cellset(mdx, **kwargs)
+        cellset_id = self.create_cellset(mdx=mdx,sandbox_name=sandbox_name, **kwargs)
```

Comment:
There are some minor formatting issues in the file. If you press `CTRL` + `ALT` + `L` in PyCharm, it should make everything compliant.